packages/core/src/util/excludedFromInstrumentation.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+// If a process is started with `npm start` or `yarn start`, two separate processes will be started. The first is
+// `node /path/to/npm` (or `node /path/to/yarn`). This process will kick off another process with the actual application
+// (for example, `node /usr/src/app`). When
+// NODE_OPTIONS="--require /usr/src/app/node_modules/@instana/collector/src/immediate" is set, we would instrument both
+// processes, that is, npm/yarn as well as the actual application. Attempting to instrument the npm or yarn process has
+// no value and also creates confusing log output, so we exclude them here explicitly.
+const excludeList = [
```

Comment:
As a personal preference, I would use "or", so the array would no longer be necessary. Unless I missed something:
```javascript
const excludePattern = /^.*\/(yarn|npm)$/;
```

Also, I added `$` to restrict the pattern to the exact name `npm` and `yarn`. The way it is, something like `yarna` or `npmagic` will not pass.

Hope it makes sense!

Reply:
Yep, that absolutely makes sense. I've changed that accordingly.